core/consensus/babe/src/lib.rs

```diff
@@ -85,7 +193,7 @@ pub trait CompatibleDigestItem: Sized {
 	fn as_babe_seal(&self) -> Option<BabeSeal>;
 }
 
-impl<Hash> CompatibleDigestItem for generic::DigestItem<Hash, Public, Secret> {
+impl<Hash: core::fmt::Debug> CompatibleDigestItem for generic::DigestItem<Hash, Public, primitives::sr25519::Signature> {
```

Comment:
Since the BABE code is all new, we'll be using the generic `DigestItem::Consensus` everywhere. That means we can make this code work `for<T> generic::DigestItem<Hash, Public, T>`, right?

Reply:
Also, >100char lines need wrapping